DBZ-6371 Use Debezium build number attribute in Dockerfile examples (2.1 branch)

roldanbob opened the pull request:
[DBZ-6371](https://issues.redhat.com/browse/DBZ-6371)

This change provides more usable examples in the downstream legacy Dockerfile-based deployment instructions by replacing the formatted text that was used to represent the build numbers with the debezium-build-number attribute that is used in the Streams deployment instructions.

The change renders in the downstream content only and has no effect on the community version of the content.
Tested in a local downstream build and merged to the downstream docs repository in [MR 1221](https://gitlab.cee.redhat.com/red-hat-integration-documentation/integration/-/merge_requests/1221).

Cherrypicks PR #4482  from main

Naros commented on documentation/modules/ROOT/pages/connectors/sqlserver.adoc:
```diff
@@ -2247,6 +2247,10 @@ To match the name of a column, {prodname} applies the regular expression that yo
 That is, the specified expression is matched against the entire name string of the column; it does not match substrings that might be present in a column name. +
 If you include this property in the configuration, do not also set the `column.include.list` property.
 
+|[[sqlserver-property-skip-messages-without-change]]<<sqlserver-property-skip-messages-without-change, `+skip.messages.without.change+`>>
+|`false`
+| Specifies whether to skip publishing messages when there is no change in whitelisted columns. This would essentially filter messages if there is no change in columns included as per `column.include.list` or `column.exclude.list` properties.
+
```
This was only applicable to Debezium 2.2, so this should be removed.

Naros commented on documentation/modules/ROOT/pages/connectors/mysql.adoc:
```diff
@@ -2663,6 +2663,10 @@ To match the name of a column, {prodname} applies the regular expression that yo
 That is, the specified expression is matched against the entire name string of the column; it does not match substrings that might be present in a column name. +
 If you include this property in the configuration, do not set the `column.exclude.list` property.
 
+|[[mysql-property-skip-messages-without-change]]<<mysql-property-skip-messages-without-change, `+skip.messages.without.change+`>>
+|`false`
+| Specifies whether to skip publishing messages when there is no change in whitelisted columns. This would essentially filter messages if there is no change in columns included as per `column.include.list` or `column.exclude.list` properties.
+
```
This was only applicable to Debezium 2.2, so this should be removed.

Naros reviewed (changes requested):
Just one comment inline @roldanbob

roldanbob commented:
@Naros Hmm. I saw the conflict after I cherry-picked the commit to my local 2.1-based branch, and I removed the `skip.messages` properties before I pushed. Looking in the repo, I don't see the property listed in the 2.1 branch, so I think that I addressed the discrepancy. Let me know if you see otherwise.

jpechane commented:
@Naros Is this good to be merged now?

roldanbob commented:
@jpechane I thought that I had merged this, hence my comment that the `skip.messages` properties were correctly withheld from the 2.1 branch. But looking now at the content on my remote fork, I see that the properties are there. I'll make the necessary corrections and commit them, and then we should be set.

roldanbob commented:
> Just a few comments inline @roldanbob .

I've addressed the comments now, @Naros, so I think that this is now ok to merge.

jpechane commented:
@roldanbob Applied, thanks
